inetum-realjobs-frontend/src/app/components/resume/details/experience-list/experience-list.component.ts

```diff
@@ -0,0 +1,25 @@
+import {Component, OnInit, Input, Output, EventEmitter} from '@angular/core';
+import {Experience} from "../../../../models/experience";
+
+@Component({
+  selector: 'app-experience-list',
+  templateUrl: './experience-list.component.html',
+  styleUrls: ['./experience-list.component.scss']
+})
+export class ExperienceListComponent implements OnInit {
+
+  @Input() experienceList: Experience[];
+
+  @Input() deleteButton: boolean;
+  @Output() deleteEvent = new EventEmitter<number>();
+
+  constructor() {
+  }
+
+  ngOnInit(): void {
+  }
```

Comment:
Empty `ngOnInit` should be removed. Don't forget to remove `implements OnInit` for the class.